onnxmltools/convert/common/_topology.py

```diff
@@ -6,6 +6,7 @@
 
 import re
 from distutils.version import StrictVersion
+from ...utils.metadata_props import add_metadata_props, color_space_to_pixel_format, set_denotation
```

Comment:
Please move `from ...utils.metadata_props import add_metadata_props, color_space_to_pixel_format, set_denotation` to line 12 (right after `from ...proto import helper`).